fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvTablet.java

```diff
@@ -345,7 +345,16 @@ public LogAppendInfo putAsLeader(
                         // should not append.
                         if (appendedRecordCount > 0) {
                             // now, we can build the full log.
-                            return logTablet.appendAsLeader(walBuilder.build());
+                            LogAppendInfo logAppendInfo =
+                                    logTablet.appendAsLeader(walBuilder.build());
+                            long logEndOffset = logAppendInfo.lastOffset();
+                            if (logEndOffset != logOffset) {
+                                LOG.warn(
+                                        "The log end offset {} is not equal to the expected log offset {}.",
+                                        logEndOffset,
+                                        logOffset);
```

Comment:
Do we still need this warning log?

Reply:
Forget to remove these code lines for debug...